src/ifcparse/Ifc2x3enum.h

```diff
@@ -29,7 +29,7 @@
 #ifndef IFC2X3ENUM_H
 #define IFC2X3ENUM_H
 
-#include "../ifcparse/IFC_PARSE_API.h"
+#include "../ifcparse/ifc_parse_api.h"
```

Comment:
This should go here: https://github.com/IfcOpenShell/IfcOpenShell/blob/master/src/ifcexpressparser/templates.py#L58 When refactoring we should probably introduce a `generated/` folder to prevent these small oversights